js/kuna.js

```diff
@@ -143,4 +143,34 @@ module.exports = class kuna extends acx {
         }, params));
         return this.parseTrades (response, market);
     }
+
+    parseMyTrades (trades, market = undefined) {
+        return Object.values (trades).map (trade => this.parseMyTrade (trade, market))
```

Comment:
This one might require a for loop, until we add support for .map to the transpiler, again... Sorry, I know, it can be frustrating sometimes.

Reply:
There's one advantage of for loops over `map()` syntax sugar, however... In general, counter-based iteration is 5-10 times faster than a `map()` of function calls. That becomes significant and noticeable if you have to do thousands of rounds of parsing every second. It's not theoretical, we are rewriting our close-source code because of that.

Reply:
> There's one advantage of for loops over map() syntax sugar, however... In general, counter-based iteration is 5-10 times faster than a map() of function calls.

Why do you do those `map`s then? (Personally, I prefer loops but have taken this snippet from the Exchange class for consistency).

> Basically, the spacing is correct, just need to remove the space inside consequent `({` and `})`

If there were a linter rules for that we'd be able to check it before transpilation.

Reply:
> Why do you do those maps then? (Personally, I prefer loops but have taken this snippet from the Exchange class for consistency).

I was lazy and the base class is not transpiled. But I'll rewrite to for loops there, thank for reminding me of that.

> If there were a linter rules for that we'd be able to check it before transpilation.

I hope to add them asap, thx for that advice as well.